index.html

```diff
@@ -20,6 +20,12 @@
     <!-- Add to home screen for Windows -->
     <meta name="msapplication-TileImage" content="<%= htmlWebpackPlugin.files.publicPath %>static/img/icons/msapplication-icon-144x144.png">
     <meta name="msapplication-TileColor" content="#000000">
+
+    <!-- Add Roboto font and Google icons -->
+    <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Roboto:300,400,500,700,400italic">
+    <link rel="stylesheet" href="//fonts.googleapis.com/icon?family=Material+Icons">
```

Comment:
Adding roboto font & material icons for use with material design. Overall size of material icons ~54KB. Could consider importing specific icons we wish to use to reduce this (as we are importing ~900 icons currently)